docs/mimxrt/pinout.rst

```diff
@@ -30,26 +30,28 @@ MIMXRT1170-EVK     Debug USB      D0/D1       D12/D11      D10/D13
 Adafruit Metro M7     -           D0/D1        D7/D3        A1/A0
 Olimex RT1010Py       -          RxD/TxD       D7/D8        D5/D6
 Seeed ARCH MIX        -         J3_19/J3_20  J4_16/J4_17  J4_06/J4_07
+Makerdiary RT1010     -           D9/D10      D13/A0       D11/D12
```

Comment:
Should this be RT1011?  There are also other cases of RT1010 in the docs here, should they all be RT1011?

Reply:
Right. While the MCU is labeled MIMXRT1011, the board names use either 1010 or 1011. NXP and Olimex use 1010, Makerdiary uses 1011. In the pinout document, the name should be short and clearly identify the board. That's why I used this form instead of i.MX RT1011 Nano Kit. I'll change it to `Makerdiary RT1011`.